test/contour.c

```diff
@@ -11,15 +11,16 @@
 #include "magics_api.h"
 #include <stdio.h>
 
+#define NUM_FORMATS 3
```

Comment:
Why did you have to do this one ?

Reply:
Well apparently MSVC doesn't support C99 - which means that it doesn't consider `const int num_formats = 3` to actually be a constant. It seems a bit mad really. Then, if `num_formats` isn't constant, the code won't compile, because MSVC also doesn't support variable length arrays. See [this commit](https://github.com/ecmwf/magics/pull/3/commits/26d1622946b86b8542366b925a5992a6ec016129) for an example fix.

We could have fixed `contour.c` in a similar way, but defining a macro seemed slightly more straightforward!

Reply:
That's what we thought ! and I had missed the change in MagicsCalls.cc